workflows/argo/resolve-output-url.yaml

```diff
@@ -16,9 +16,14 @@ spec:
         command: [bash]
         source: |
           set -e
+          remove_slashes () {
+              echo $(echo $1 | sed -e 's~/~-~')
```

Comment:
I think it's probably better to fail if there is a `/` rather than transform the user inputs. One nice thing about vcm-workflow-control is that we can use `git-grep` to find commits matching a specific url. This feature won't work if we transform the user inputs.

Reply:
OK